taxjar/client.py

```diff
@@ -164,9 +168,19 @@ def _request(self, method, endpoint, data=None):
     def _uri(api_url, endpoint):
         return api_url + endpoint
 
+    @staticmethod
+    def _getUserAgent():
```

Comment:
@ScottRudiger Use snake_case for this method name, e.g. `_get_user_agent`

Reply:
Oops, good catch!